website/docs/rules/intl/prefer-provide-intl-description.mdx

```diff
@@ -0,0 +1,83 @@
+import RuleDetails from '@site/src/components/RuleDetails';
+
+<RuleDetails version="5.5.0" severity="warning" />
+
+Warns when a `Intl.message()`, `Intl.plural()`, `Intl.gender()` or `Intl.select()` method invocate without description.
```

Comment:
Warns when the .... methods are invoked without a description.

Reply:
fixed